api/rest/mantisbt_openapi.yaml

```diff
@@ -0,0 +1,1733 @@
+          schema:
+            type: integer
+            minimum: 1
+          description: ''
+          required: true
+          example: 5
+      requestBody:
+        $ref: '#/components/requestBodies/project_delete_user'
+  '/api/rest/users':
+    post:
+      summary: Create a user
+      operationId: userAdd
+      description: ''
+      tags:
+        - 'Users'
+      responses:
+        '200':
+          description: 'Success'
+        '4XX':
+          description: 'Bad Request'
+      requestBody:
+        $ref: '#/components/requestBodies/create_a_user'
+  '/api/rest/users/{id}':
+    get:
+      summary: Get User By Id
+      operationId: userGet
+      description: ''
+      tags:
+        - 'Users'
+      responses:
+        '200':
+          description: 'Success'
+        '4XX':
+          description: 'Bad Request'
+      parameters:
+        - name: id
+          in: path
+          schema:
+            type: integer
+          description: ''
+          required: true
+          example: 1
+        - name: select
+          in: query
+          schema:
+            type: string
+          description: ''
+          required: false
+          example: id,name,real_name,email,access_level
+  '/api/rest/users/username/{username}':
+    get:
+      summary: Get User By Username
+      operationId: userGetByUsername
+      description: ''
+      tags:
+        - 'Users'
+      responses:
+        '200':
+          description: 'Success'
+        '4XX':
+          description: 'Bad Request'
+      parameters:
+        - name: username
+          in: path
+          schema:
+            type: string
+          description: ''
+          required: true
+        - name: select
+          in: query
+          schema:
+            type: string
+          description: ''
+          required: false
+          example: id,name,real_name,email
+  '/api/rest/users/{user_id}/reset':
+    put:
+      summary: Reset user password
+      operationId: userResetPassword
+      description: ''
+      tags:
+        - 'Users'
+      responses:
+        '200':
+          description: 'Success'
+        '4XX':
+          description: 'Bad Request'
+      parameters:
+        - name: user_id
+          in: path
+          schema:
+            type: integer
+            minimum: 1
+          description: ''
+          required: true
+          example: 5
+  '/api/rest/users/{user_id}':
+    patch:
+      summary: Update User
+      operationId: userUpdate
+      description: ''
+      tags:
+        - 'Users'
+      responses:
+        '200':
+          description: 'Success'
+        '4XX':
+          description: 'Bad Request'
+      parameters:
+        - name: user_id
+          in: path
+          schema:
+            type: integer
+            minimum: 1
+          description: ''
+          required: true
+      requestBody:
+        $ref: '#/components/requestBodies/update_user'
+servers:
+  - url: https://mantisbt.org/bugs/
```

Comment:
Add server entry to describe the user's instance,

https://github.com/dregad/mantisbt/blob/7199f739cf7cf9b116a918522d1c03f45efe25e3/api/rest/openapi.yaml#L29-L47